### PR TITLE
add(msg): number of commits to diff

### PR DESCRIPTION
**Pull request by chdsbd**
We now include the number of commits and the number of lines added and removed.

### CODE REVIEW

File: .env-example [hunk 1]
```diff
@@ -1,3 +1,9 @@
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_REGION=us-east-1
+TTD_GITHUB_APP_ID=59400
+TTD_GITHUB_INSTALL_ID=143423402
+TTD_GITHUB_APP_PRIVATE_KEY_BASE_64=QTBBNUU5RUItQkRGNC00OEI3LTkzRTktNUExMjhEMDg3OUM3Cg==
```
Comment (chdsbd) [inline .env-example:6]: this isn't a real key

File: src/index.ts [hunk 2]
```diff
@@ -36,9 +40,18 @@ export async function handler(event: unknown) {
...
+      privateKeyBase64: env.right.TTD_GITHUB_APP_PRIVATE_KEY_BASE_64,
+    }),
     db: createDbClient(env.right.TTD_DYNAMO_TABLE_NAME),
     env: env.right,
     event,
     getCurrentDate,
   })
 }
+
+if (require.main === module) {
```
Comment (sbdchd) [inline src/index.ts:55]: what's this for?

File: src/message.ts [hunk 3]
```diff
@@ -289,12 +304,28 @@ async function getStagingSha({
...
+    repo: string
+    base: string
+    head: string
+  }) => Promise<Comparison | null>
+}
+
+function getOrgRepo({ url }: { readonly url: string }) {
+  // https://github.com/ghost/example/ -> [ghost, example]
+  const [org, repo] = new URL(url).pathname.split("/").filter(Boolean)
+  return [org, repo] as const
```
Comment (sbdchd) [inline src/message.ts]:
```suggestion
  return {org, repo}
```

File: src/index.ts (hunk 2 above)
Comment (chdsbd) [reply src/index.ts:55]: so I can run the index.js locally

**Review by sbdchd — APPROVED** [review]

**Pull request merged**